openedx/features/edly/tests/utils.py

```diff
@@ -0,0 +1,30 @@
+from django.conf import settings
+from django.test.client import RequestFactory
+
+from rest_framework.test import APIClient
+
+from openedx.core.djangoapps.site_configuration.tests.factories import SiteFactory
+from openedx.features.edly import cookies
+from openedx.features.edly.tests.factories import EdlySubOrganizationFactory
+
+def get_edly_middleware_authorized_client(enforce_csrf_checks=False, domain=None, client_class=None):
```

Comment:
I am not seeing the usage of this method. Can we remove it?

Also the name can be simplified like`get_edly_client`

Reply:
removed